packager/react-packager/index.js

```diff
@@ -118,6 +117,16 @@ function createNonPersistentServer(options) {
   return createServer(options);
 }
 
+function omit(obj, blacklistedKeys) {
```

Comment:
@grabbou that would mean that we need to extract this function somewhere. I'd like to do it, but I have no clue where to put it. Any suggestions?